mlir/include/mlir/Dialect/MIOpen/gridwise_gemm_params.h

```diff
@@ -608,6 +615,8 @@ class PopulateParams : public PopulateParamsBase {
       obtainGemmCVecLen(ctx, outputVecLen);
     }
 
+    outputVecLen = std::__gcd(outputVecLen, param->gemmNPerThread);
```

Comment:
What's the difference between this std::__gcd(), and the gcd defined in the beginning of this file?